dashboard/final-example/app/ui/invoices/create-form.tsx

```diff
@@ -44,17 +44,20 @@ export default function Form({ customers }: { customers: CustomerField[] }) {
             <UserCircleIcon className="pointer-events-none absolute left-3 top-1/2 h-[18px] w-[18px] -translate-y-1/2 text-gray-500" />
           </div>
 
-          {state.errors?.customerId ? (
-            <div
-              id="customer-error"
-              aria-live="polite"
-              className="mt-2 text-sm text-red-500"
-            >
-              {state.errors.customerId.map((error: string) => (
-                <p key={error}>{error}</p>
-              ))}
-            </div>
-          ) : null}
+          <div
```

Comment:
I agree using div is probably more semantically correct than using spans.
This being said, I'm not sure then that we should add the className="mt-2 text-sm text-red-500" directly on this div element ?
Because it creates a small space underneath the label, even when there is no error to be shown.

I know it's not ideal but maybe consider using a div inside a div ?

`                     <div aria-live='polite' aria-atomic='true' id='customer-error'>
                        {state.errors?.customerId ? (
                            <div  className='mt-2 text-sm text-red-500'>
                                {state.errors.customerId.map((error: string) => (
                                    <p key={error}>{error}</p>
                                ))}
                            </div>
                        ) : null}
                    </div>`

There might be some more elegant ways, I don't know.
Or the extra space is simply not that problematic.

Reply:
Thank you for the suggestion, agree. Committed a solution to make it more tight, we can remove the fragment and move the classes to the `<p>`.